### DISCUSSION TITLE
Multiple Profiles to track habits of family members as well.

### DISCUSSION THREAD

**Discussion by mujul** [iSoron/uhabits#842] [Feature Requests]
Sir your app is simple and great. I want to track my habits as well as those of my family members. A feature to create and manage isolated profile can be helpful. An option where i can create new profile and everything is brand new as well as isolated from the previous profile.

**Comment by hiqua** [comment]
What about creating habits in the same (unique) "profile" and just have a prefix, or a different color for other people?

This seems particularly niche, I'd argue that most users who want to use this app have their own phone.

**Reply by Mrnofish** [reply]
> This seems particularly niche, I'd argue that most users who want to use this app have their own phone.

There are multiple use cases in which "buy yourself a phone" is not a feasible answer: babies, people with certain disabilities, just to name two.

And it's not very niche: while this suggestion is worded very specifically, it can be seen in the larger perspective of an existing demand for more sophisticated managing of habits.

That said, the request does not need to be implemented literally to be satisfied: **habit grouping** and/or **tagging** could work.

**Comment by romain030992** [comment]
Hello! 

Great feature! It would be even better to manage accounts so people of the same family can "share" habits. I'm thinking for example about cleaning tasks that can be completed by everybody in the family.
